modules/ROOT/pages/index-vm-bare-metal.adoc

```diff
@@ -85,7 +85,7 @@ The successful operation of Anypoint Runtime Fabric on VMs / Bare Metal is a sha
 
 The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
 
-image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsiblities]
+image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsibilities]
```

Comment:
Generally, don't just repeat the name of the image in the alt text. Is this decorative or something? Then [""]